mit_frontend/src/routers/UserInfo.js

```diff
@@ -8,50 +8,72 @@ import DropDownMenu from 'material-ui/DropDownMenu';
 import MenuItem from 'material-ui/MenuItem';
 import RadioButton from 'material-ui/RadioButton';
 import ArrowDropRight from 'material-ui/svg-icons/navigation-arrow-drop-right';
-import '../App.css';
+import '../App.css'; 
 import styles from './styles';
+import AutoComplete from 'material-ui/AutoComplete';
+import Geosuggest from 'react-geosuggest';
+import SelectField from 'material-ui/SelectField';
+import ReactDOM from 'react-dom';
 
-// Initiate Job Year
-const jobYears = [];
-for (let i = 0; i <= 50; i += 1) {
-  jobYears.push(<MenuItem value={i} key={i} primaryText={i} />);
+/* Need this to circumvent ESlint when using Google API */
+/*global google*/
+
+/* Initiate Job Year */
+const jobYear = [];
```

Comment:
I feel like we should make `jobYear` and `gradYear` `TextField`s, but I don't have too strong of opinion on this.